webshare/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        setupManualShareTest(false, '', '', location.toString());
```

Comment:
The way the spec is written, the the document's base URL should be used to resolve the URL, can you use `document.baseURI` here? Even better would be a test where the base URL and the URL are not the same, to catch bugs where the wrong thing is used, but take that as an optional suggestion.